.github/workflows/run_tests.yaml

```diff
@@ -112,6 +112,12 @@ jobs:
         export GSLDIR=$(gsl-config --prefix)
         export PYTHONPATH=$(pwd):$PYTHONPATH
         NuRadioMC/test/SignalGen/test_build.sh
+    - name: "Validate separate trigger channels"
+      if: always()
+      run: |
+        export GSLDIR=$(gsl-config --prefix)
```

Comment:
is that line actually needed or is that a copy past relict?

Reply:
Probably not. I don't fully understand how the workflows work, so I just copied and pasted. Which lines specifically are not needed?

Reply:
the export